minet/cli/twitter/timestamp.py

```diff
@@ -0,0 +1,28 @@
+# =============================================================================
+# Minet Twitter Tweets CLI Action
+# =============================================================================
+#
+# Logic of the `tw tweets` action.
+#
+import casanova
+
+from minet.cli.utils import LoadingBar
+from twitwi.utils import get_timestamp_from_id
+
+
+def twitter_timestamp_action(cli_args):
+
+    enricher = casanova.enricher(
+        cli_args.file,
+        cli_args.output,
+        keep=cli_args.select,
+        add=["timestamp"],
```

Comment:
I think that the column should be called "timestamp_utc", to be consistent with current twitwi column names: https://github.com/medialab/twitwi/blob/master/twitwi/constants.py#L14

Reply:
good call